### PR TITLE
docs: add Admin API samples for account management methods

### PR DESCRIPTION
**Pull request by ikuleshov**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-analytics-admin/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by tmatsuo** [comment]
> There are 7 possible violations for not having product prefix.

The snippet-bot thinks the product prefix is wrong.
Can you register the product in the devrel service?

File: samples/account_summaries_list.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google Inc. All Rights Reserved.
```
Comment (tmatsuo) [inline samples/account_summaries_list.py:3]: The copyright holder should be `Google LLC`

File: samples/account_summaries_list.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+all accounts accessible by the caller.
+
+See https://developers.google.com/analytics/devguides/config/admin/v1/rest/v1alpha/accountSummaries/list
+for more information.
+"""
+# [START analyticsadmin_account_summaries_list]
+from google.analytics.admin import AnalyticsAdminServiceClient
+
+
+def run_sample():
```
Comment (tmatsuo) [inline samples/account_summaries_list.py:27]: What's the purpose of this function?

File: samples/account_summaries_list.py (hunk 1 above)
Comment (tmatsuo) [reply samples/account_summaries_list.py:3]: And all the other files too.

**Review by tmatsuo — CHANGES_REQUESTED** [review]
noxfile.py seems old. Can you copy it from:
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/noxfile-template.py
?

in noxifle_config.py, I think you want to change `enforce_type_hints` to False.

@dandhlee 
@leahecole Just in case I'm not saying something wrong.

The code seems good, but can you take care of:
- license holder names
- snippet-bot warning
- copy the latest noxfile.py
- turn off enforce_type_hints

Thanks

**Comment by leahecole** [comment]
We typically do request that type annotations be enforced for new Python samples, so it would be preferred to keep "enforce type hints" on and add [type hints](https://www.python.org/dev/peps/pep-0484/) to the functions. 

Regarding the latest noxfile for samples, copying the template totally works, but is there a reason why it's not picking up changes from the template in synthtool? 

You can also remove the `noxfile_config_test.py` file 

> noxfile.py seems old. Can you copy it from:
> https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/noxfile-template.py
> ?
> 
> in noxifle_config.py, I think you want to change `enforce_type_hints` to False.
> 
> @dandhlee
> @leahecole Just in case I'm not saying something wrong.
> 
> The code seems good, but can you take care of:
> 
> * license holder names
> * snippet-bot warning
> * copy the latest noxfile.py
> * turn off enforce_type_hints
> 
> Thanks

File: samples/account_summaries_list.py (hunk 2 above)
Comment (ikuleshov) [reply samples/account_summaries_list.py]: Removed occurrences of  run_samples() where the code is trivial.

File: samples/account_summaries_list.py (hunk 1 above)
Comment (ikuleshov) [reply samples/account_summaries_list.py]: Done!

**Comment by ikuleshov** [comment]
Added type hints, with enforce_type_hints=True.

**Comment by ikuleshov** [comment]
The proper region tag prefix [seems to be set up](https://screenshot.googleplex.com/476BctttVuXVBpG), but the test still insists it is not.

**Comment by tmatsuo** [comment]
@ikuleshov The product is registered, but marked as private, so data file doesn't have the prefix yet. I'm ok if you ignore the failing status check.

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**